backend/src/main/kotlin/hu/bme/sch/cmsch/component/token/TokenCollectorService.kt

```diff
@@ -39,6 +39,9 @@ open class TokenCollectorService(
     open fun collectToken(user: CmschUser, token: String): TokenSubmittedView {
         val tokenEntity = tokenRepository.findAllByTokenAndVisibleTrue(token).firstOrNull()
         if (tokenEntity != null) {
+            if (!isTokenActive(tokenEntity)) {
+                return TokenSubmittedView(TokenCollectorStatus.CANNOT_COLLECT, null, null, null)
```

Comment:
I might add a new failure reason here that explains the reason better. If it is urgent we might create a new ticket out of it instead of implementing it.

Reply:
Good idea, created new status